ohsome_quality_api/indicators/attribute_completeness/indicator.py

```diff
@@ -69,28 +71,51 @@ def calculate(self) -> None:
         if self.result.value is None:
             self.result.description += " No features in this region"
             return
-        description = Template(self.templates.result_description).substitute(
-            result=round(self.result.value, 2),
-            all=round(self.absolute_value_1, 1),
-            matched=round(self.absolute_value_2, 1),
-        )
+        self.create_description()
 
         if self.result.value >= self.threshold_yellow:
             self.result.class_ = 5
             self.result.description = (
-                description + self.templates.label_description["green"]
+                self.description + self.templates.label_description["green"]
             )
         elif self.threshold_yellow > self.result.value >= self.threshold_red:
             self.result.class_ = 3
             self.result.description = (
-                description + self.templates.label_description["yellow"]
+                self.description + self.templates.label_description["yellow"]
             )
         else:
             self.result.class_ = 1
             self.result.description = (
-                description + self.templates.label_description["red"]
+                self.description + self.templates.label_description["red"]
             )
 
+    def create_description(self):
+        attribute_names = [
+            get_attribute(self.topic.key, attribute_key).name.lower()
+            for attribute_key in self.attribute_key
+        ]
+        if self.topic.aggregation_type == "count":
```

Comment:
Maybe the next 10 lines could be extracted into a method names `computeUnitsForAllAndMatched` or similar (returning a tuple of `all, matched`) in order to make the method shorter and less complex. This could also be unit-tested in isolation.